eng/Version.Details.xml

```diff
@@ -1,32 +1,32 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="8.0.100-rtm.23526.1">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="8.0.100-rtm.23530.12">
       <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>0b8e35cfcba7710d9feb86221e79faadda820c97</Sha>
+      <Sha>ae28d5bc061eedfe863ef3e22d090054b69e7685</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NETCore.App.Ref" Version="8.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>59edaad404d1b8e47080015ae8d0787f94c970df</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.0.40">
+    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.0.42">
       <Uri>https://github.com/xamarin/xamarin-android</Uri>
-      <Sha>f63ac29fb3e48bfb5785e9fad03a64f68f7fc750</Sha>
+      <Sha>5f1b293be96260dba6a2838d73237d35863d980e</Sha>
```

Comment:
This looks like the latest version here: https://github.com/xamarin/xamarin-android/commits/release/8.0.1xx